ucm/sof-glkda7219max/HiFi.conf

```diff
@@ -0,0 +1,106 @@
+SectionVerb {
+
+	EnableSequence [
+		cdev "hw:sofglkda7219max"
+	]
+
+	DisableSequence [
+		cdev "hw:sofglkda7219max"
+	]
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker Playback"
+
+	Value {
+		PlaybackPCM "hw:sofglkda7219max,0"
+		PlaybackChannels "2"
+	}
+
+	EnableSequence [
+		cdev "hw:sofglkda7219max"
+		cset "name='Spk Switch' on"
+	]
+
+	DisableSequence [
+		cdev "hw:sofglkda7219max"
+		cset "name='Spk Switch' off"
+	]
+}
+
+SectionDevice."Headphone Playback" {
+	Comment "Headphone Playback"
+
+	Value {
+		PlaybackPCM "hw:sofglkda7219max,1"
+		PlaybackChannels "2"
+	}
```

Comment:
no JackControl?

Reply:
@plbossart for HDMI we got `JackControl "HDMI/DP,pcm=7 Jack"`, but for headset, seems no such info in `amixer contents`, so headphone JackControl, and jack detection related is not present in this ucm.

Reply:
well jack detection should be added here for headphones/headset. It's supported in the machine driver so there's no reason to remove it.